pkg/render/typha.go

```diff
@@ -308,9 +316,23 @@ func (c *typhaComponent) typhaDeployment() *apps.Deployment {
 	maxUnavailable := intstr.FromInt(1)
 	maxSurge := intstr.FromString("25%")
 
+	var initContainers []v1.Container
 	annotations := make(map[string]string)
 	annotations[typhaCAHashAnnotation] = AnnotationHash(c.typhaNodeTLS.CAConfigMap.Data)
-	annotations[typhaCertHashAnnotation] = AnnotationHash(c.typhaNodeTLS.TyphaSecret.Data)
+	if c.installation.CertificateManagement == nil {
+		annotations[typhaCertHashAnnotation] = AnnotationHash(c.typhaNodeTLS.TyphaSecret.Data)
+	} else {
+		annotations[typhaCertHashAnnotation] = AnnotationHash(c.installation.CertificateManagement.CACert)
+		initContainers = append(initContainers, CreateCSRInitContainer(
+			c.installation,
```

Comment:
Do we need to pass both the `installation` as well as a subfield of the installation? (`installation.CertificateManagement`)

Maybe we could just remove the second argument?